flocker/restapi/_infrastructure.py

```diff
@@ -267,11 +267,9 @@ def loadAndDispatch(self, request, **routeArguments):
 
                 def got_result(result):
                     code = OK
-                    json = result
                     if isinstance(result, EndpointResponse):
                         code = result.code
-                        json = result.result
-                    eliot_action.add_success_fields(code=code, json=json)
```

Comment:
:question: This can be bundled with the other logging reduction changes, but I don't think this one needs a followup. Logging the JSON responses seems excessive and I don't remember it being useful while debugging acceptance tests.